js/models/fixed_page_spread.js

```diff
@@ -90,13 +128,27 @@ var Spread = function(spine, isSyntheticSpread) {
         }
     };
 
+    /**
+     * Resets the items
+     *
+     * @method     resetItems
+     */
+
     function resetItems() {
 
         self.leftItem = undefined;
         self.rightItem = undefined;
         self.centerItem = undefined;
     }
 
+    /**
+     * Sets the item to a position on the screen
+     *
+     * @method     setItemToPosition
+     * @param      item
+     * @param      position
```

Comment:
position is an enum {Spread.POSITION_CENTER | Spread.POSITION_LEFT | Spread.POSITION_RIGHT}